### DISCUSSION TITLE
信用卡付款 登录邮件没有登录验证码

### DISCUSSION THREAD

**Discussion by pppppppppppppjsjns** [getlantern/lantern-forum-cn#763] [Q&A]
充值看了剩余时间到25年10月,第二天就要重新登陆不发邮箱验证码
银行卡4065 9537 2488 6381 
帐号574471458@qq.com
您尾号6381交行信用卡05日00时06分成功消费人民币344.00

**Comment by pppppppppppppjsjns** [comment]
![Uploading 17230460010760.17969485887850645.jpg…]()

**Reply by pppppppppppppjsjns** [reply]
交易图片

**Reply by MaxMKMax** [reply]
@pppppppppppppjsjns 您好. 好的, 我帮您处理. 我这里看不到您的截图, 请您发支付单据到我的邮箱好吗? 谢谢

**Reply by pppppppppppppjsjns** [reply]
邮箱发两张图片给您了能收到么

**Comment by pppppppppppppjsjns** [comment]
---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年8月8日(周四) 凌晨1:33
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 信用卡付款 登录邮件没有登录验证码 (Discussion #763)




 
@pppppppppppppjsjns 您好. 好的, 我帮您处理. 我这里看不到您的截图, 请您发支付单据到我的邮箱好吗? 谢谢
